packages/vm/src/runCall.ts

```diff
@@ -12,7 +12,7 @@ export interface RunCallOpts {
   block?: Block
   gasPrice?: bigint
   origin?: Address
-  caller?: Address
```

Comment:
This should still be optional, can just default to zero address in `new Message`

Reply:
Fixed!